### PR TITLE
[ITSASU-1100] Remove all <br> tags and replace with margins

### PR DESCRIPTION
**Pull request by aug24**

### CODE REVIEW

File: app/views/individual/claimenrolment/ClaimEnrolmentConfirmation.scala.html [hunk 1]
```diff
@@ -62,10 +62,8 @@ <h2 class="govuk-heading-m">@messages("claimConfirm.heading")</h2>
                     </a>))
             </p>
 
-            <p class = "govuk-body">@messages("sign-up-complete.whatHappensNow.para2")</p>
-            <br>
+            <p class="govuk-body govuk-!-margin-bottom-6">@messages("sign-up-complete.whatHappensNow.para2")</p>
             @buttonHelper(messages("claimConfirm.continueButton"))
```
Comment (sb-dev) [inline app/views/individual/claimenrolment/ClaimEnrolmentConfirmation.scala.html:66]: Without `<br>` the sign out link gets displayed on the same line as the continue button:
```suggestion
            @buttonHelper(messages("claimConfirm.continueButton"), classes = Some("govuk-!-display-block"))
```

File: app/views/individual/ContinueRegistration.scala.html [hunk 2]
```diff
@@ -30,13 +30,14 @@
...
-    @continueButton(Some(messages("preferences_callback.button")))
-  }
-  <br>
-  @signOutLink()
+    @form(action = postAction) {
+        @continueButton(Some(messages("preferences_callback.button")))
+    }
+    <span class="govuk-!-display-block govuk-body govuk-!-margin-bottom-6">
+        @signOutLink()
+    </span>
```
Comment (sb-dev) [inline app/views/individual/ContinueRegistration.scala.html:42]:
```suggestion
    <div class="govuk-!-margin-bottom-6">
        @signOutLink()
    </div>
```
```suggestion
    <span class="govuk-!-display-block govuk-body govuk-!-margin-bottom-6">
        @signOutLink()
    </span>
```

Comment (sb-dev) [inline app/views/individual/ContinueRegistration.scala.html:42]:
```suggestion
    <div class="govuk-!-margin-bottom-6">
        @signOutLink()
    </div>
```
```suggestion
    <span class="govuk-!-display-block govuk-body govuk-!-margin-bottom-6">
        @signOutLink()
    </span>
```

File: scripts/start [hunk 3]
```diff
@@ -8,7 +8,7 @@ function die() {
 }
 
 MONGO_PORT=27017
-which nc &>/dev/null && nc -vz localhost $MONGO_PORT 2>/dev/null || die "Can't find MongoDB on port $MONGO_PORT (using: nc -vz localhost $MONGO_PORT)"
+which nc &>/dev/null && nc -vz localhost $MONGO_PORT 2>/dev/null || die "Can't find MongoDB on port $MONGO_PORT (using: nc -vz localhost $MONGO_PORT) - you may want to run 'docker run -p 27017:27017 -d mongo'"
```
Comment (aug24) [inline scripts/start:11]: Just a little nice to have. Always good to tell people the fix.

File: project/FrontendBuild.scala [hunk 4]
```diff
@@ -68,7 +68,7 @@ private object AppDependencies {
         "org.scalatest"                %% "scalatest"            % scalaTestVersion     % scope,
         "com.typesafe.play"            %% "play-test"            % PlayVersion.current  % scope,
         "org.scalatestplus.play"       %% "scalatestplus-play"   % scalaTestPlusVersion % scope,
-        "com.github.fge"               % "json-schema-validator" % "2.2.14"             % scope,
+        "com.github.fge"               % "json-schema-validator" % "2.2.6"             % scope,
```
Comment (aug24) [inline project/FrontendBuild.scala:71]: I've hit this on several repos.  Not sure where the 2.2.14 release came from or has disappeared to!

**Review by sb-dev — APPROVED** [review]

**Pull request merged**